src/Api/Controllers/CiphersController.cs

```diff
@@ -398,6 +418,24 @@ public async Task PutDeleteMany([FromBody]CipherBulkRestoreRequestModel model)
             await _cipherService.SoftDeleteManyAsync(model.Ids.Select(i => new Guid(i)), userId);
         }
 
+        [HttpPut("delete-admin")]
+        public async Task PutDeleteManyAdmin([FromBody]CipherBulkRestoreRequestModel model)
+        {
+            if (!_globalSettings.SelfHosted && model.Ids.Count() > 500)
+            {
+                throw new BadRequestException("You can only restore up to 500 items at a time.");
```

Comment:
missed copy+paste issue, "delete" vs. "restore" in exception message.

Reply:
That verbiage is actually pulled from the PutDeleteMany method. They both also use a CipherBulkRestoreRequestModel as the request body. I would say this might actually be more of a naming issue than an issue with the error message, but I'm not 100%.

Reply:
Yes, because the action is delete, let's change the verbiage in the exception message here (the model name is a misnomer there).

Reply:
Updated. Also went ahead and changed the model to the correct one & updated the non-admin methods.